src/modules/database/entities/data-model-item-relation.entity.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import DataModelItem from './data-model-item.entity';
+
+@Entity()
+export default class DataModelItemRelation extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ length: 500, nullable: false})
```

Comment:
Why do you need this if you have id of item?

Reply:
Model item = table in database
Model item field = some column in this table that could be not included in the model item itself.

For example, we will have no Id field in some Model item, but we have relations depends on this field in the real table, so first_model_item_field will be 'id', first_model_item_id will be the id of some model item to get the original name of the table.